patches/net/minecraft/client/renderer/ItemBlockRenderTypes.java.patch

```diff
@@ -117,6 +127,9 @@
 +
 +    private static net.neoforged.neoforge.client.ChunkRenderTypeSet createSetFromPredicate(java.util.function.Predicate<RenderType> predicate) {
 +        return net.neoforged.neoforge.client.ChunkRenderTypeSet.of(RenderType.chunkBufferLayers().stream().filter(predicate).toArray(RenderType[]::new));
-     }
- 
++    }
++
++    // Neo: Disable cutout mode on FAST rendering
```

Comment:
Oh man lol. Because this patch started with FORGE START but had no FORGE END, I thought setFancy was a patch we did lmao. More reason to yeet the start and end comments. Will fix shortly

Reply:
Ok so I removed this comment and moved out patches in this file to be below setFancy. That way after a certain point, it's just neo additions which would be easier to understand